src/components/SessionList.vue

```diff
@@ -130,9 +137,15 @@ export default {
 		},
 		sessionStyle() {
 			return (session) => {
-				return {
-					'border-color': session.color,
-					'background-color': session.color + ' !important',
+				if (this.showAuthorAnnotations) {
+					return {
+						border: 'solid 2px ' + session.color + ' !important',
+					}
+				} else {
+					return {
+						border: 'solid 2px var(--color-main-background) !important',
+						'background-color': session.color + ' !important',
+					}
```

Comment:
We can avoid else here.

```suggestion
				if (this.showAuthorAnnotations) {
					return {
						border: 'solid 2px ' + session.color + ' !important',
					}
				}
			    return {
				  border: 'solid 2px var(--color-main-background) !important',
				  'background-color': session.color + ' !important',
			    }
```

Reply:
Sure